lib/python/picongpu/pypicongpu/grid.py

```diff
@@ -101,4 +111,19 @@ def _get_serialized(self) -> dict:
                 "y": self.n_gpus[1],
                 "z": self.n_gpus[2],
             },
+            "super_cell_size": {
+                "x": self.super_cell_size[0],
+                "y": self.super_cell_size[1],
+                "z": self.super_cell_size[2],
+            },
         }
+        if self.grid_dist is not None:
+            result_dict |= {
+                "grid_dist": {
+                    "x": list(map(lambda x: {"device_cells": x}, self.grid_dist[0])),
```

Comment:
Non-blocking but some style guides would advice you to prefer
```suggestion
                    "x": [{"device_cells": x} for x in self.grid_dist[0]]
```

You could even reduce the whole to
```python
{direction: [{"device_cells": x} for x in dist] for direction, dist in zip("xyz", self.grid_dist)}
```
This general form could also be helpful once we're allowing 2D.